install-panel.sh

```diff
@@ -136,35 +136,16 @@ required_input() {
 }
 
 password_input() {
-  local  __resultvar=$1
-  local  result=''
-
-  while [ -z "$result" ]; do
-    echo -n "* ${2}"
-
-    # modified from https://stackoverflow.com/a/22940001
-    while IFS= read -r -s -n1 char; do
-      [[ -z $char ]] && { printf '\n'; break; } # ENTER pressed; output \n and break.
-      if [[ $char == $'\x7f' ]]; then # backspace was pressed
-          # Only if variable is not empty
-          if [ -n "$result" ]; then
-            # Remove last char from output variable.
-            [[ -n $result ]] && result=${result%?}
-            # Erase '*' to the left.
-            printf '\b \b' 
-          fi
-      else
-        # Add typed char to output variable.
-        result+=$char
-        # Print '*' in its stead.
-        printf '*'
-      fi
+  local result=$1
+  echo -en "* ${2}\n"
+  while [[ -z "$password_main" || ! "$password_main" == "$password_confirm" ]]
```

Comment:
`password_main` is a rather vague variable name, can you precise it further?

Reply:
do you have a suggestion?

Reply:
I am not too sure. I'd suggest sticking with the same format as the `required_input` function.